.buildkite/pipeline.full.yml

```diff
@@ -73,13 +73,13 @@ steps:
       queue: opensource
     plugins:
       artifacts#v1.5.0:
-        download: ["features/fixtures/ios/output/iOSTestApp.ipa"]
+        download: "features/fixtures/ios/output/ipa_url.txt"
```

Comment:
This approach allows the app to be uploaded to BrowserStack just once for a CI run (the text file just includes a unique reference for the upload).  The change was made some time ago, but got missed on this particular step.